plugins/providers/hyperv/driver.rb

```diff
@@ -218,6 +218,68 @@ def set_name(vmname)
         execute(:set_name, VMID: vm_id, VMName: vmname)
       end
 
+      #
+      # Disk Driver methods
+      #
+
+      # @param [String] controller_type
+      # @param [String] controller_number
+      # @param [String] controller_location
+      # @param [Hash] opts
+      def attach_disk(disk_file_path,  **opts)
+        execute(:attach_disk_drive, VmId: @vm_id, Path: disk_file_path, ControllerType: opts[:ControllerType],
+                ControllerNumber: opts[:ControllerNumber], ControllerLocation: opts[:ControllerLocation])
+      end
+
+      # @param [String] path
+      # @param [Int] size_bytes
+      # @param [Hash] opts
+      def create_disk(path, size_bytes, **opts)
+        # ensure size_bytes is a uint64
```

Comment:
Is a type check required here? Or if this is a note on why it's a separate parameter, maybe relocate it to the doc block above with a little context.

Reply:
This was mostly a note to myself that I missed when cleaning up the PR.